Assets/Scripts/CrumblingFloorEvent.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CrumblingFloorEvent : MonoBehaviour
+{
+    public float crumbleDelay;
+
+    public AudioClip crumbling;
+
+    public AudioClip crumblingWarning;
+
+    private float timerCountDown = 1;
+
+    bool isColliding = false;
+
+    bool hasCrumblingAudioPlayed = false;
+
+
+    private void Update()
+    {
+        if (isColliding == true)
```

Comment:
Sorry that I wasn't more specific but the rocks should crumble if collided even if you're no longer on it anymore. Right now you can jump over and over and have the platform not disappear.
For example, if I just on the platform and then jump off of it, it should still disappear after the given time